src/dom.js

```diff
@@ -19,11 +21,7 @@ function updateProperty (dom, name, value, newValue) {
 export function updateElement (dom, props, newProps) {
   Object.keys(newProps)
     .filter(isNew(props, newProps))
-    .forEach(key => {
-      key === 'value' || key === 'nodeValue'
-        ? (dom[key] = newProps[key])
-        : updateProperty(dom, key, props[key], newProps[key])
-    })
```

Comment:
No special handling of `value` and `nodeValue` required anymore, as these are properties - they will be detected and handled by the `name in dom` test above.

Reply:
I can reduce code sizes here 😄